src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,125 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int GROW_FACTOR = 2;
+    private int size;
+    private Node<K, V>[] table;
+    private Node<K, V> nullKeyNode;
```

Comment:
```suggestion
```

Reply:
Give me a comment please. I can't figure out what I need to do

Reply:
remove this field and try to implement your solution without it